assets/prometheus-k8s/rules.yaml

```diff
@@ -866,6 +866,14 @@ spec:
     - expr: sum(cluster:master_nodes or on(node) kube_node_labels ) BY (label_beta_kubernetes_io_instance_type,
         label_node_role_kubernetes_io)
       record: cluster:node_instance_type_count:sum
+    - expr: sum  by (provisioner) (kube_persistentvolumeclaim_resource_requests_storage_bytes
+        * on (namespace,persistentvolumeclaim) group_right() (kube_persistentvolumeclaim_info
+        * on (storageclass)  group_left(provisioner) kube_storageclass_info))
+      record: cluster:storage_provisioner_capacity_requested:total
+    - expr: sum  by (provisioner) (kubelet_volume_stats_used_bytes * on (namespace,persistentvolumeclaim)
+        group_right() (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner)
+        kube_storageclass_info))
+      record: cluster:storage_provisioner_capacity_used:total
```

Comment:
Recording rules have certain naming rules see docs for this https://prometheus.io/docs/practices/rules/

`cluster:kubelet_volume_stats_used_bytes:provisioner:sum`

 would be more appropriate here

Reply:
Same goes with the other recording rule.

Reply:
ack, I will update the PR, Thanks

Reply:
@lilic I have made the changes.